cpp/src/Ice/UdpEndpointI.cpp

```diff
@@ -15,10 +15,32 @@ using namespace std;
 using namespace Ice;
 using namespace IceInternal;
 
+namespace
+{
+    const char* const udpPluginName = "IceUDP";
+}
+
 extern "C"
```

Comment:
Here this is only for static libraries, extern C seems not relevant. You either use a dynamic Ice build and this is loaded, or you use a static library and you have to use the factory to register it. I assuming extern "C" here is only relevant for dlsym and similar mechanism, where we want to avoid name mangling. That applies to TPC/WS too.